Python/Product/Django.Analysis/DjangoAnalyzer.cs

```diff
@@ -198,17 +198,22 @@ internal static class Commands {
             analyzer.SpecializeFunction("django.contrib.gis.shortcuts", "render_to_kmz", RenderToStringProcessor, true);
             analyzer.SpecializeFunction("django.contrib.gis.shortcuts", "render_to_text", RenderToStringProcessor, true);
 
-            analyzer.SpecializeFunction("django.template.base.Library", "filter", FilterProcessor, true);
-            analyzer.SpecializeFunction("django.template.base.Library", "filter_function", FilterProcessor, true);
+            analyzer.SpecializeFunction("django.template.Library", "filter", FilterProcessor, true);
+            analyzer.SpecializeFunction("django.template.Library", "filter_function", FilterProcessor, true);
 
-            analyzer.SpecializeFunction("django.template.base.Library", "tag", TagProcessor, true);
-            analyzer.SpecializeFunction("django.template.base.Library", "tag_function", TagProcessor, true);
-            analyzer.SpecializeFunction("django.template.base.Library", "assignment_tag", TagProcessor, true);
-            analyzer.SpecializeFunction("django.template.base.Library", "simple_tag", TagProcessor, true);
+            analyzer.SpecializeFunction("django.template.Library", "tag", TagProcessor, true);
+            analyzer.SpecializeFunction("django.template.Library", "tag_function", TagProcessor, true);
+            analyzer.SpecializeFunction("django.template.Library", "assignment_tag", TagProcessor, true);
+            analyzer.SpecializeFunction("django.template.Library", "simple_tag", TagProcessor, true);
```

Comment:
This works for both because when `django.template` is imported, it'll import `Library` from the correct module and then overwrite the member. So even if you import it directly from `django.template.library.Library`, you still get the specialization.